# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/tiles/reftest.list
# Commit: f733b37f9626
# Full Hash: f733b37f9626548ae83564a3fd43ddfa972977d9
# Author: Nicolas Silva <nsilva@mozilla.com>
# Date: 2021-04-22 21:31:57
# Regressor Bug: 1702228
# File Overlap Count: 1
# Description:
#   Bug 1702228 - Adjust reftests. r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D112970
# ==============================================================================

diff -r 0919633e25e9 -r f733b37f9626 gfx/wr/wrench/reftests/tiles/reftest.list
--- a/gfx/wr/wrench/reftests/tiles/reftest.list	Thu Apr 22 10:34:55 2021 +0000
+++ b/gfx/wr/wrench/reftests/tiles/reftest.list	Thu Apr 22 10:34:55 2021 +0000
@@ -1,5 +1,5 @@
 ** rect.yaml
-** simple-gradient.yaml
+fuzzy(1,2000) ** simple-gradient.yaml
 # TODO: Fix rasterizer inaccuracies so this is the same regardless of tile size!
 !* prim-suite.yaml
 == mix-blend-clip.yaml mix-blend-clip-ref.yaml